.github/workflows/backport.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+# A workflow to automatically backport pull requests to a stable branch.
+#
+# This workflow uses the Backport CLI (https://github.com/sorenlouv/backport)
+# under the hood, which is configured in `.backportrc.json` in the repository
+# root.
+#
+# See https://github.com/sorenlouv/backport-github-action for documentation
+# on the used action.
+
+name: Backport PRs to stable branches
+
+on:
+  pull_request_target:
+    # Run this workflow when a label on a PR is added, or if it's closed.
+    types: ["labeled", "closed"]
```

Comment:
This would run if an open PR is labeled, before it's accepted, and then you would have to rebase and force push changes after the fact if there were any. Is there a way to prevent the backport from opening until the PR is merged.

Also is there a more specific target than "closed" for "merged"?

Reply:
The trigger here is too broad, because we cannot express the exact trigger we need. There's further narrowing happening below in the `if` clause. Ultimately, the backport PRs are only opened after the source PR is merged.

Reply:
Just making sure it's only opened after it is merged, and not if it's closed (not accepted or abandoned).